tox.ini

```diff
@@ -97,7 +97,16 @@ ignore =
     D10,
     # Single vs double quotes conflicting with black
     Q000,
-
+black-config = pyproject.toml
+exclude =
+    .git,
+    __pycache__,
+    .tox,
+    .env,
+    .venv,
+    env,
+    venv,
+    */node_modules
```

Comment:
Why add this here if pyproject.toml is configuring black already?

Reply:
I think undoing the changes to this file should fix the CI tests, since they are throwing the error "duplicate key exclude"

Reply:
the issue with the 'duplicate key exclude' was an uncaught duplicate from Roni's change. It has been fixed. I did remove the reference to pyproject.toml because it worked either way.

Reply:
@waxlamp  18e1f9fca0e18439fe96718fa219d7fa8c5b9093